fs_gcsfs/_gcsfs.py

```diff
@@ -32,9 +33,11 @@ class GCSFS(FS):
     Args:
         bucket_name: The GCS bucket name.
         root_path: The root directory within the GCS Bucket
+        create: Whether to create ``root_path`` on initialization or not. If ``root_path`` does not yet exist and ``create=False`` an ``CreateFailed``
```

Comment:
Minimal suggestion: I think it should be rather "... **a** CreateFailed exception" than "... **an** CreateFailed exception".